onthefly/backend/redis_backend.py

```diff
@@ -14,6 +14,9 @@ def set(self, name, value):
 
     def get(self, name):
         dumped = self.client.hget(self.bucket_prefix, name)
+        if not dumped:
+            print(f'onthefly warning: {name} is None')
```

Comment:
This would require Python 3.6. I can use ordinary print format if you want to support lower Python, or use Python logging.